Clockwise/VirtualClock.cs

```diff
@@ -60,10 +60,16 @@ private VirtualClock(DateTimeOffset? now = null, [CallerMemberName] string creat
                     {
                         break;
                     }
+                    var shouldStop = nextTime == time;
```

Comment:
If the user requested AdvanceTo(TimeUntilNextActionIsDue), then only execute one action.

Reply:
So that can result in, effectively, `AdvanceBy(TimeSpan.Zero)`. That's a bit confusing. It's unintuitive that it would trigger only a single action if multiple are scheduled for the same instant. Maybe that messiness should be encapsulated by something a bit more intention-revealing, e.g. `AdvanceToNextAction`.

Reply:
I have one practical objection and one theoretical objection

* Right now, the ServiceBus clockwise implementation expects *only* one action to be executed on an invocation to `AdvanceTo(TimeUntilNextActionIsDue)`.  
* I think that `AdvanceBy(Timespan.Zero)` is perfectly fine.  Clockwise is built around the concept that the only thing that consumes time is the time in between schedulings.

If you would prefer to make the Service Bus not rely on the first bullet-point, I'm happy to budge on bullet-point two.

Reply:
Put differently, I think that it's not clear what the expected behavior should be when one advances the clock by `TimeSpan.Zero`, and a method that performs exactly one action is useful and clearer.